.github/workflows/ci.yml

```diff
@@ -95,6 +95,22 @@ jobs:
       - name: Run Check
         run: yarn ${{ matrix.lint-task }}
 
+  stylelint:
+    name: Stylelint
+    needs: [install]
+    runs-on: ubuntu-tatest
+    steps:
+      - name: Checkout
+        actions: actions/checkout@v3
+      - name: Install
+        uses: ./.github/actions/prepare-install
+        with:
+        node-version:
+          node-version: ${{ env.PRIMARY_NODE_VERSION }}
+      - name: Run stylelint check
+        run: yarn stylelint
+        working-directory: website
```

Comment:
🤦 I shouldn't have approved this - I'd touched up the code locally, got distracted, then forgot I did that. Then didn't notice that it failed in CI.

Just a heads up @juank1809 - this needed a few changes.

```diff
  stylelint:
    name: Stylelint
    needs: [install]
-    runs-on: ubuntu-tatest
+    runs-on: ubuntu-latest
    steps:
      - name: Checkout
-        actions: actions/checkout@v3
+        uses: actions/checkout@v3
      - name: Install
        uses: ./.github/actions/prepare-install
        with:
          node-version: ${{ env.PRIMARY_NODE_VERSION }}
      - name: Run stylelint check
        run: yarn stylelint
-        working-directory: website
+        working-directory: packages/website
```

Next time I'd suggest forking this repo and testing the action out in your fork. And on my end I'll do a better job reviewing. 😄